valve-monitor/web-app/src/services/AppModel.ts

```diff
@@ -12,6 +13,12 @@ export interface Project extends DomainModel.Project {
   devices: Device[] | null; // null when devices are not required
 }
 
+export interface ValveMonitorConfig {
+  monitorFrequency?: number;
+  minFlowThreshold?: number;
+  maxFlowThreshold?: number;
```

Comment:
I wonder if these should include the word `fleet` in them somewhere so we can distinguish between fleet level and device level env vars. I'm not sure if that will be useful or not to us